server.go

```diff
@@ -2020,6 +2020,70 @@ func (s *Server) executeShowTagValuesStatement(stmt *influxql.ShowTagValuesState
 	return result
 }
 
+func (s *Server) executeGrantStatement(stmt *influxql.GrantStatement, user *User) *Result {
+	s.mu.RLock()
+	defer s.mu.RUnlock()
+
+	// Look up the user in the statement that will be granted the privilege.
+	// NOTE: the user passed in by the caller is the granter.
+	u, ok := s.users[stmt.User]
+	if !ok {
+		return &Result{Err: ErrUserNotFound}
+	}
+
+	// Check if this privilege is being granted on the cluster.
+	if stmt.On == "" {
+		// The only privilege allowed on the cluster is admin (AllPrivileges).
+		if stmt.Privilege != influxql.AllPrivileges {
+			return &Result{
+				Err: fmt.Errorf("cannot grant %s on the cluser, only %s",
+					stmt.Privilege.String(),
+					influxql.AllPrivileges.String()),
+			}
+		}
+
+		// Grant user cluster admin privileges.
+		u.Admin = true
```

Comment:
I might be issuing something, but this doesn't do anything to update the cluster. It simply updates the in-RAM user object, right? Restart this node, or hit a different node, and this change is not there.